core/services/ocr2/plugins/functions/types.pb.go

```diff
@@ -31,7 +31,7 @@ type Query struct {
 func (x *Query) Reset() {
 	*x = Query{}
 	if protoimpl.UnsafeEnabled {
-		mi := &file_core_services_ocr2_plugins_functions_types_proto_msgTypes[0]
+		mi := &file_types_proto_msgTypes[0]
```

Comment:
hmm this looks weird but maybe just the new proto gen version has different naming scheme... integration tests should confirm that nothing got broken.

Reply:
Yeah I am not quite sure, but if its not breaking anything, it should be fine. Right?